### PR TITLE
Whats new: Adds note for improved quota visibility from whats new presentation

### PR DESCRIPTION
**Pull request by opayne1**
Relates to [Console-922](https://issues.redhat.com/browse/CONSOLE-922) and [Slide in What's New Presentation](https://docs.google.com/presentation/d/1wu9yfHU2XbDksFKakatHDvFjVIY307f1au29NwdUNds/edit#slide=id.g1144d9dd107_14_922)

- Applies to `enterprise-4.10` only
- [Preview link](https://deploy-preview-42071--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-web-console)

### CODE REVIEW

**Comment by opayne1** [comment]
@yapei PTAL. Thank you!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -174,6 +174,10 @@ Additionally, the pod status pop-up window now provides links to the *Logs* and
 ==== Customized workload notifications
 With this update, you can customize workload notifications on the *User Preferences* page. *User workload notifications* under the *Notifications* tab allows you to hide user workload notifications that appear on the *Cluster Overview* page or in your drawer.
 
+[id="improved-quota-visibility-web-console"]
+==== Improved Quota Visibility
+With this update, non-admin users are now able to view their usage of the `AppliedClusterResourceQuota` on the *Project Overview*, *ResourceQuotas*, and *API Explorer* pages in order to determine the cluster-scoped quota available for use. Additionally, `AppliedClusterResourceQuota` details can now able to be found on the *Search* page.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:179]: can now be found on

**Comment by ahardin-rh** [comment]
Just one comment from me! :+1:

**Pull request merged**